tests/src/python/test_qgssettings.py

```diff
@@ -224,6 +224,32 @@ def test_global_groups(self):
         self.assertEqual([], self.settings.globalChildGroups())
         self.settings.endGroup()
 
+    def test_group_section(self):
+        # Test group by using Section
+        # Reset firstgroup first, to make sure that there is no value before the test
+        self.settings.remove('firstgroup', section=QgsSettings.Core)
```

Comment:
this is not necessary: each test uses a different settings file, see `setUp` https://github.com/ismailsunni/QGIS/blob/18523db8d8d3cf66349291688d24e36fb9a60041/tests/src/python/test_qgssettings.py#L37

Reply:
Hi @elpaso 
Thanks for the review. I have removed the lines in an ammend commit.